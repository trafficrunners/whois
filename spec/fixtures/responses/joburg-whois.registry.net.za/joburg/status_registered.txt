Domain Name: usedautos.joburg
Domain ID: dom_7P-9999
WHOIS Server: joburg-whois2.registry.net.za

Updated Date: 2014-11-10T07:08:28Z
Creation Date: 2014-11-03T22:00:08Z
Registry Expiry Date: 2015-11-03T22:00:08Z
Sponsoring Registrar: Lexsynergy
Sponsoring Registrar IANA ID: 1466 
Domain Status: clientDeleteProhibited
Domain Status:clientTransferProhibited
Domain Status:clientUpdateProhibited

Registrant ID: LEX-5FP-22YL
Registrant Name: Domain Administrator
Registrant Organization: The Car Trader (Pty) Ltd
Registrant Street: 154 Bram Fischer Drive Randburg 
Registrant City: Johannesburg
Registrant State/Province: 
Registrant Postal Code: 2194
Registrant Country: ZA
Registrant Phone: +27.116860900
Registrant Phone Ext: 
Registrant Fax: +27.117896449
Registrant Fax Ext: 
Registrant Email: domains@autotrader.co.za

Admin ID: LEX-5FP-22YL
Admin Name: Domain Administrator
Admin Organization: The Car Trader (Pty) Ltd
Admin Street: 154 Bram Fischer Drive Randburg 
Admin City: Johannesburg
Admin State/Province: 
Admin Postal Code: 2194
Admin Country: ZA
Admin Phone: +27.116860900
Admin Phone Ext: 
Admin Fax: +27.117896449
Admin Fax Ext: 
Admin Email: domains@autotrader.co.za

Billing ID: LEX-1-PGD
Billing Name: Domain Name Department
Billing Organization: Lexsynergy Limited
Billing Street: 130 Hampstead House 176 Finchley Road 
Billing City: London
Billing State/Province: 
Billing Postal Code: NW3 6BT
Billing Country: GB
Billing Phone: +44.2081331319
Billing Phone Ext: 
Billing Fax: +44.2081331319
Billing Fax Ext: 
Billing Email: domains@lexsynergy.com

Tech ID: LEX-5FP-22YL
Tech Name: Domain Administrator
Tech Organization: The Car Trader (Pty) Ltd
Tech Street: 154 Bram Fischer Drive Randburg 
Tech City: Johannesburg
Tech State/Province: 
Tech Postal Code: 2194
Tech Country: ZA
Tech Phone: +27.116860900
Tech Phone Ext: 
Tech Fax: +27.117896449
Tech Fax Ext: 
Tech Email: domains@autotrader.co.za


Name Server: ns1.lexsynergy.net
Name Server: ns2.lexsynergy.us
Name Server: ns3.lexsynergy.info
DNSSEC: unsigned
>>> Last update of WHOIS database: 2015-06-02T09:11:02Z <<<

#    WHOIS lookup made at 2015-06-02T09:11:02Z
# --
# The use of this Whois facility is subject to the following terms and
# conditions. https://registry.net.za/whois_terms
# Copyright (c) ZACR 1995-2015
