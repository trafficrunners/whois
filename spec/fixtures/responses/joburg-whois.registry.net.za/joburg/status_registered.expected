#domain
  %s == "usedautos.joburg"

#domain_id
  %s == "dom_7P-9999"


#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2014-11-03 22:00:08 UTC}

#updated_on
  %s %CLASS{time}
  %s %TIME{2014-11-10 07:08:28 UTC}

#expires_on
  %s %CLASS{time}
  %s %TIME{2015-11-03 22:00:08 UTC}


#registrar
  %s %CLASS{registrar}
  %s.id           == "1466"
  %s.name         == "Lexsynergy"
  %s.organization == "Lexsynergy"
  %s.url          == nil

#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_REGISTRANT
  %s[0].id            == "LEX-5FP-22YL"
  %s[0].name          == "Domain Administrator"
  %s[0].organization  == "The Car Trader (Pty) Ltd"
  %s[0].address       == "154 Bram Fischer Drive Randburg"
  %s[0].city          == "Johannesburg"
  %s[0].zip           == "2194"
  %s[0].country       == nil
  %s[0].country_code  == "ZA"
  %s[0].phone         == "+27.116860900"
  %s[0].fax           == "+27.117896449"
  %s[0].email         == "domains@autotrader.co.za"
  %s[0].created_on    == nil
  %s[0].updated_on    == nil

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_ADMINISTRATIVE
  %s[0].id            == "LEX-5FP-22YL"
  %s[0].name          == "Domain Administrator"
  %s[0].organization  == "The Car Trader (Pty) Ltd"
  %s[0].address       == "154 Bram Fischer Drive Randburg"
  %s[0].city          == "Johannesburg"
  %s[0].zip           == "2194"
  %s[0].country       == nil
  %s[0].country_code  == "ZA"
  %s[0].phone         == "+27.116860900"
  %s[0].fax           == "+27.117896449"
  %s[0].email         == "domains@autotrader.co.za"
  %s[0].created_on    == nil
  %s[0].updated_on    == nil

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Record::Contact::TYPE_TECHNICAL
  %s[0].id            == "LEX-5FP-22YL"
  %s[0].name          == "Domain Administrator"
  %s[0].organization  == "The Car Trader (Pty) Ltd"
  %s[0].address       == "154 Bram Fischer Drive Randburg"
  %s[0].city          == "Johannesburg"
  %s[0].zip           == "2194"
  %s[0].country       == nil
  %s[0].country_code  == "ZA"
  %s[0].phone         == "+27.116860900"
  %s[0].fax           == "+27.117896449"
  %s[0].email         == "domains@autotrader.co.za"
  %s[0].created_on    == nil
  %s[0].updated_on    == nil


#nameservers
  %s %CLASS{array}
  %s %SIZE{3}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns1.lexsynergy.net"
  %s[0].ipv4 == nil
  %s[0].ipv6 == nil
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns2.lexsynergy.us"
  %s[1].ipv4 == nil
  %s[1].ipv6 == nil
  %s[2] %CLASS{nameserver}
  %s[2].name == "ns3.lexsynergy.info"
  %s[2].ipv4 == nil
  %s[2].ipv6 == nil
